### PR TITLE
Update AppsensorUiApplication.java

### PR DESCRIPTION
**Pull request by SriHarshaGajavalli**
Added Forbidden Error page which helps in case of login opportunity is not available

### CODE REVIEW

**Comment by jtmelton** [comment]
@SriHarshaGajavalli thanks very much for the PR! 

The code so far looks good, but it will break the application. A couple of changes need to be made: 
1. The link to the error page is to "/4.3.html", when it should be "/403.html"
2. The 403.html page does not actually exist, and will need to be created and added to the PR (this can be modeled after the other pages (400, 401, etc))

If you can take care of those 2 things, I'll be happy to merge this.

**Comment by SriHarshaGajavalli** [comment]
Ok. I'll add both

**Comment by jtmelton** [comment]
@SriHarshaGajavalli thanks for the effort here - really appreciate it!

File: appsensor-ui/src/main/java/org/owasp/appsensor/AppsensorUiApplication.java [hunk 1]
```diff
@@ -59,7 +59,7 @@ public EmbeddedServletContainerCustomizer containerCustomizer(){
         public void customize(ConfigurableEmbeddedServletContainer container) {
         	ErrorPage error400Page = new ErrorPage(HttpStatus.BAD_REQUEST, "/400.html");
             ErrorPage error401Page = new ErrorPage(HttpStatus.UNAUTHORIZED, "/401.html");
-	    ErrorPage error403Page = new ErrorPage(Httpstatus.FORBIDDEN, "/4.3.html");
+	    ErrorPage error403Page = new ErrorPage(Httpstatus.FORBIDDEN, "/403.html");
```
Comment (vallihalldors) [inline appsensor-ui/src/main/java/org/owasp/appsensor/AppsensorUiApplication.java]: This doesn't compile, it should be HttpStatus.FORBIDDEN but not Httpstatus.FORBIDDEN

**Comment by jtmelton** [comment]
Thanks @vallihalldors - I'll fix that shortly.

**Comment by jtmelton** [comment]
@vallihalldors can you take a quick peek? this should be resolved now - https://github.com/jtmelton/appsensor/blob/master/appsensor-ui/src/main/java/org/owasp/appsensor/AppsensorUiApplication.java

**Comment by vallihalldors** [comment]
@jtmelton looks good :)

**Comment by SriHarshaGajavalli** [comment]
Actually it was a typo.
Anyways thanks for both of you for finding and correcting it

**Pull request merged**